Si strip clusters digis per fed id rebased

Wilsker opened the pull request:
Re-based from previous pull request:
Adding histograms to for Num. of clusters/digis per fed ID for Sistrip FED monitoring #19000
https://github.com/cms-sw/cmssw/pull/19000

Functionality added to SiStripMonitorClusters and SiStripMonitorDigis packages to enable monitoring of the number of Clusters or number of digis per FED Id.

SiStripMonitorClusters package edits:

    Monitoring element instantiated in SiStripMonitorClusters.h
    Histogram PSet created in SiStripMonitorClusters_cfi.py
    NumberOfFEDClus Histogram added to SiStripMonitorClusters.cc in "createMEs" method.
    Map of FED Id and associated cumulative sum of clusters from all detectors and layers created.
    Loop over detector Ids, get Fed Id and add number of clusters to cumulative sum.
    Edit map values for given fedid.
    Use map to fill histogram.
    All the above in "analyze" method of SiStripMonitorClusters.cc.

SiStripMonitorDigi package edits:

    Monitoring element instantiated in SiStripMonitorDigis.h
    Histogram PSet created in SiStripMonitorDigis_cfi.py
    NumberOfFEDigis Histogram added to SiStripMonitorDigis.cc in "createMEs" method.
    Map of FED Id and associated cumulative sum of digis from all detectors and layers created.
    Loop over detector Ids, get Fed Id and add number of digis to cumulative sum.
    Edit map values for given fedid.
    Use map to fill histogram.
    All the above in "analyze" method of SiStripMonitorDigis.cc.

Commit:
Add efficiency plots vs. eta and phi for cut on discriminators

Commit:
add base tmb base classes

Commit:
change name

Commit:
put in correct class

Commit:
new structure of LUTs

Commit:
do not use CSCTriggerGeometry

Commit:
distinguish between l1 and l2

Commit:
cleanup generator class

Commit:
cleanup

Commit:
define chamber parity

Commit:
separate function for rpc from csc detid

Commit:
suggestions by Kevin Pedro

Commit:
cleanup

Commit:
suggestions by Kevin Pedro

Commit:
reduce code duplication

cerminar committed:
handle the RAW data-tier in a special way to avoid using the event-content definition of the release which is not enough for this special stream

jshlee committed:
change gemsegments to etaPartitions

jshlee committed:
change GEMMuon matching to etaPartitions

jshlee committed:
revert to superchamber gemsegments

rekovic committed:
Merge branch 'mydevel_eta_correction_grid' of https://github.com/gkaratha/cmssw into gkaratha-mydevel_eta_correction_grid

rekovic committed:
Merge branch 'gkaratha-mydevel_eta_correction_grid' into l1t-integration-CMSSW_9_2_0

rekovic committed:
Merge branch 'mydevel_eta_lut_restoration' of https://github.com/gkaratha/cmssw into gkaratha-mydevel_eta_lut_restoration

rekovic committed:
Merge branch 'gkaratha-mydevel_eta_lut_restoration' into l1t-integration-CMSSW_9_2_0

rekovic committed:
Merge branch 'metEmulatorSaturation' of https://github.com/bundocka/cmssw into bundocka-metEmulatorSaturation

rekovic committed:
Merge branch 'bundocka-metEmulatorSaturation' into l1t-integration-CMSSW_9_2_0

rekovic committed:
Merge branch 'L1EG_caloParams_2017_v1_8_2' of https://github.com/tstreble/cmssw into tstreble-L1EG_caloParams_2017_v1_8_2

rekovic committed:
Merge branch 'tstreble-L1EG_caloParams_2017_v1_8_2' into l1t-integration-CMSSW_9_2_0

rekovic committed:
Merge branch 'CaloL2_PUTowerThresh' of https://github.com/tstreble/cmssw into tstreble-CaloL2_PUTowerThresh

rekovic committed:
Merge branch 'tstreble-CaloL2_PUTowerThresh' into l1t-integration-CMSSW_9_2_0

davignon committed:
Merged l1t-integration-v95.7 from repository cms-l1t-offline

rekovic committed:
Define MC L1TReEmul workflows with using re-emulated EcalTPs (L1TReEmulMCFromRAWSimEcalTP), re-emulated HcalTPs (L1TReEmulMCFromRAWSimHcalTP), and both EcalTPs & HcalTPs re-emulated (L1TReEmulMCFromRAWSimCalTP)

jalimena committed:
fix

davignon committed:
Adding dummy L1T Tau trimming and shape veto LUT

mmusich committed:
SiStrip Payload inspector revamp

rekovic committed:
Set default caloParams 2017_v1_8.

Commit:
Divide efficiencies: protect against low stats

rekovic committed:
Merge branch 'l1t-integration-v95.7-CMSSW_9_2_0-TrimmingLUT3' of https://github.com/davignon/cmssw into davignon-l1t-integration-v95.7-CMSSW_9_2_0-TrimmingLUT3

rekovic committed:
Merge branch 'davignon-l1t-integration-v95.7-CMSSW_9_2_0-TrimmingLUT3' into l1t-integration-CMSSW_9_2_0

mmusich committed:
adding history plots for Gain payload inspector

makortel committed:
Remove pixel tracking from phase2, add firstStepPrimaryVertices

bundocka committed:
Merged l1t-integration-v95.10 from repository cms-l1t-offline

mmusich committed:
add method to Histogram1D analogous to TH1::SetBinContent

mmusich committed:
add averages per barrel layer

rekovic committed:
Revert "Merge branch 'bundocka-metEmulatorSaturation' into l1t-integration-CMSSW_9_2_0"

This reverts commit ab3e78d22a0d26fd7086021bbe129d843fe2557c, reversing
changes made to c881514ac7adba3d725387e092852bbb2f4f92fe.

rekovic committed:
Use caloParams 2017_v1_7.

rekovic committed:
Use caloParams 2017_v1_8.

bundocka committed:
Fix MET saturation in demux emulator

rekovic committed:
Revert "Revert "Merge branch 'bundocka-metEmulatorSaturation' into l1t-integration-CMSSW_9_2_0""

This reverts commit 0ceec2da01a418e3cfd07728dbbeea28f5696158.

rekovic committed:
Merge branch 'metDemuxSaturationFix' of https://github.com/bundocka/cmssw into bundocka-metDemuxSaturationFix

rekovic committed:
Merge branch 'bundocka-metDemuxSaturationFix' into l1t-integration-CMSSW_9_2_0

rekovic committed:
Use caloParams 2017_v1_7.

rekovic committed:
Use caloParams 2017_v1_8.

jaylawhorn committed:
Add Razor trigger monitoring to offline DQM

jaylawhorn committed:
remove hard-coded values, un-needed collections; currently has runtime error

jaylawhorn committed:
fix runtime error; add comments

jaylawhorn committed:
fix typo

jshlee committed:
only me0seeding

jshlee committed:
revert gemseg

jshlee committed:
revert gemseg

jshlee committed:
revert gem geo

jshlee committed:
Merge branch 'GEMSeeding-92x' into GEMSeeding-91x

jshlee committed:
revert gemseg matching

jaylawhorn committed:
fix naming conflict with HLTOffline validation for razor triggers (relval 1306.0 successful)

bundocka committed:
Merged l1t-integration-v95.12 from repository cms-l1t-offline

Commit:
Address comments

mmusich committed:
fix FillByBinAndValue to mimick TH1::SetBinContent

mmusich committed:
add plots for Endcaps

mmusich committed:
add y-axis label in Histogram1D

mmusich committed:
Fix handling of DetIds

jaylawhorn committed:
fixes from mtosi comments

fioriNTU committed:
adding a simple track cut

Commit:
Correct mistyping

fioriNTU committed:
implementing David suggestions

fabozzi committed:
adding premixing PU 2017 workflows

Commit:
Tried to address magic # issue

jalimena committed:
updated consumes, changed ints to unsigned ints, made GenericTriggerEventFlags unique_ptrs, moved structs inside class definition

mmusich committed:
small bug-fix in plot_SiStripApvEndcapPlusGainsByDisk

rekovic committed:
Add ReEmul version L1TReEmulMCFrom90xRAWSimHcalTP, suitable for 90x MC samples which contain simHcalUnsuppressedDigis.

fioriNTU committed:
Change some plots to TProfiles and add 5 histos for OnTrack Clusters (5*1600=8000  bins added)

makortel committed:
Fix initialStep

jaylawhorn committed:
update to include comments from davidlange6

slehti committed:
HLTTauDQMOfflineSource: changed bool tagAndProbe -> tagAndProbe_

slehti committed:
HLTTauDQMOfflineSource: bugfix

bundocka committed:
Add caloParams, towETLUTs & LUT generator, update customiseSettings for excluding ieta=30 from METHF

bundocka committed:
Now update customiseSettings

bundocka committed:
Add caloParams 1_9 without 0.7 HF SF for JEC

rekovic committed:
Merge branch 'excludeEta30FromMETHF' of https://github.com/bundocka/cmssw into bundocka-excludeEta30FromMETHF

rekovic committed:
Merge branch 'bundocka-excludeEta30FromMETHF' into l1t-integration-CMSSW_9_2_0

Commit:
suggestions by David Lange

mmusich committed:
make tidier loops on detids and use DetId::subdetId

Commit:
ME0 pad digi clusters

Commit:
enable ME0 pad digi clusters

Commit:
use clusters to build triggers

Commit:
add comments

slehti committed:
hltTauOfflineMonitor_TagAndProbe: added monitoring for the lepton legs of the lepton+tau triggers

rekovic committed:
Address comments: Use LogError and remove non-needed map clearing in the destructor.

rekovic committed:
Address comments: Make more robust skipping of lines in the LUT file.

rekovic committed:
Address comments: document magic numbers 69 and 117, and remove commented lines.

rekovic committed:
Address comments: remove commented lines.

rekovic committed:
Address comments: Use LogError.

Commit:
Take David's comments

Commit:
move declusterization to TMB

Commit:
add partition number

Commit:
Use array instead of tuple

rekovic committed:
Addressing comments: simplify string+int assignment, make skip of commented lines in LUT more robust, reserve vector to a known size before pushing.

rekovic committed:
Fix introduced bug when deriving LUT file name.

Commit:
Fix || operator usage in conditional statements

This resolves 4 warnings from GCC 7.1.1 where we most likely have a
wrong usage of || operator in our conditional statements.

Signed-off-by: David Abdurachmanov <David.Abdurachmanov@cern.ch>

Commit:
Naming convention

Commit:
Move int* to vector

jalimena committed:
change guard to DQMOFFLINE_TRIGGER_NOBPTXMONITOR_H, change nbins to unsigned int

rekovic committed:
Fix introduced bug when skiping lines, remove additional lines not starting with #.

fabozzi committed:
use of early 2017 BS gensim for premix 2017 wf

slehti committed:
HLTTauDQMOffline_cfi: bugfix

Commit:
Moved analyzers to smart pointers

rekovic committed:
Fix introduced bug when skiping lines, remove additional lines not starting with #.

davignon committed:
Merged l1t-integration-v95.14 from repository cms-l1t-offline

davignon committed:
Fix of EG and Tau clustering filtering

bpfrancis committed:
Update to 2017 METplusTrack paths

jalimena committed:
remove some histograms and change some binning

jaylawhorn committed:
update from comments from davidlange6; drop histograms vs LS

davignon committed:
Updated tau merging to not consider secondaries with TT ieta>30

Commit:
max 8 clusters per ME0

davignon committed:
undo last commit

Commit:
Can access front layer position

Commit:
Record particle flux in selected volumes

lantone committed:
second rebase attempt

Commit:
helper function

Teemperor committed:
Add include for AlgebraicObjects.h to HIPUserVariables.h

We use AlgebraicSymMatrix in this header, so we also need to
include the header that provides this class to make this file
compile on it's own.

jalimena committed:
fix

slehti committed:
hltTauOfflineMonitor_TagAndProbe: changed the phi plot region from -3.14to3.14 to -3.15to3.15

Teemperor committed:
Add missing include for DOMNode.hpp to MiscalibReaderFromXMLDomUtils.h

The MiscalibReaderFromXMLDomUtils.icc file we include in this file
tries to access the getNodeValue() method of a DOMNode, so we also
need to include the related header to make this header compile
on it's own.

Teemperor committed:
Foward declare TH1F in APVGainStruct.h

We have a pointer to TH1F in this header, but we don't have any
includes. This causes this files not to compile as a C++ module.
This patch just adds the missing forward declaration to TH1F to
fix this.

Teemperor committed:
Add missing include to HITRegionalPixelSeedGenerator.h

This header uses IsolatedPixelTrackCandidateCollection, so we also
need to include the related header to make this file compile on
it's own.

threus committed:
update GT for online DQM to v4

lantone committed:
also remove obsolete paths from config files

fioriNTU committed:
possible fix to failing tests

Commit:
Modify the logic to suit David's taste

Commit:
Tune the mismatch Ratio QT thresholds to 0 for L1T Muons

Commit:
Update the L1TStage2RatioClient to be able to ignore bins

rekovic committed:
Merged l1t-integration-v95.15 from repository cms-l1t-offline

Conflicts:
	L1Trigger/L1TMuon/data

davignon committed:
Fixes to caloNavigation

rekovic committed:
Merge branch 'l1t-integration-v95.14-CMSSW_9_2_0-ClusterFilteringFix' of https://github.com/davignon/cmssw into davignon-l1t-integration-v95.14-CMSSW_9_2_0-ClusterFilteringFix

rekovic committed:
Merge branch 'davignon-l1t-integration-v95.14-CMSSW_9_2_0-ClusterFilteringFix' into l1t-integration-CMSSW_9_2_0

rekovic committed:
Merged l1t-integration-v95.16 from repository cms-l1t-offline

jaylawhorn committed:
update for revised trigger list

Wilsker committed:
Rebased to from-CMSSW_9_2_X_2017-06-16-1100

slehti committed:
hltTauOfflineMonitor_TagAndProbe: changed single electron path name

pieterdavid committed:
Remove tracker subdetector DetId in some Reco/Geometry ESProducts

- PFCheckHitPattern in RecoParticleFlow/PFTracking
- SimpleDAFHitCollector in RecoTracker/SiTrackerMRHTools
- TkGluedMeasurementDet in RecoTracker/MeasurementDet

ianna committed:
Remove double includes

Wilsker committed:
rebased to 9_2_X

rekovic committed:
Address comments: Replace isChosen() with hasattr( ).

rekovic committed:
Address comments: simplify logic in CaloStage2Nav.h

rekovic committed:
Address comments: remove commented lines of temporary solution.

Commit:
suggestions by David Lange

Commit:
suggestions by David Lange

davidlange6 committed:
add tests for downhill,theanets,xgboost

davidlange6 committed:
add also hep_ml and uncertainties

pieterdavid committed:
PFCheckHitPattern: use TrackerTopology::layer

also removed PFCheckHitPattern::interpretDetId

Commit:
SiStripApvGain write to its tags (and not multirun)

rekovic committed:
Address comment: document expert level customisation calo param config file.

mmusich committed:
Adding the DQMEventInfo module in SiStripGains and SiPixelAli PCL applications

Commit:
add SiStripApvGainRcdAAG_multirun

Commit:
include SiStripApvGainRcdAAG_multirun in writing script

Commit:
content of SiStripApvGainRcdAAG_multirun

Commit:
suggestions by Kevin Pedro

Commit:
suggestions by Kevin Pedro

Commit:
update include statements

Commit:
update include statements

Commit:
update include statements

z4027163 committed:
Remove stage2 modules from Legacy L1Trigger DQMOffline Sequence

davidlange6 committed:
Update runtests.sh

fix dumb typo

Commit:
change auto to const auto& in several places as suggested

Commit:
Following coding rules

sarafiorendi committed:
add possibility to avoid matching to the L1 firing L1 seed

cmsbuild commented:
A new Pull Request was created by  @Wilsker  for master.

It involves the following packages:

DQM/SiStripMonitorCluster
DQM/SiStripMonitorDigi


@vazzolini, @kmaeshima, @dmitrijus, @cmsbuild, @vanbesien, @davidlange6 can you please review it and eventually sign? Thanks.
@idebruyn, @threus, @fioriNTU, @hdelanno this is something you requested to watch as well.
@davidlange6 you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

ianna committed:
Merge and resolve conflicts

cmsbuild committed:
Merge pull request #19377 from davidlange6/pythonunits2

add tests for downhill,theanets,xgboost

cmsbuild committed:
Merge pull request #19275 from Teemperor/FixIncludesHITRegionalPixelSeedGenerator.h

Add missing include to HITRegionalPixelSeedGenerator.h

cmsbuild committed:
Merge pull request #19272 from Teemperor/FixIncludesMiscalibReaderFromXMLDomUtils.h

Add missing include for DOMNode.hpp to MiscalibReaderFromXMLDomUtils.h

cmsbuild committed:
Merge pull request #19268 from Teemperor/FixIncludesHIPUserVariables.h

Add include for AlgebraicObjects.h to HIPUserVariables.h

Teemperor committed:
Fix typo in OrPairSelector.h

cmsbuild committed:
Merge pull request #19273 from Teemperor/ForwardDeclareTH1FinAPVGainStruct.h

Foward declare TH1F in APVGainStruct.h

cmsbuild committed:
Merge pull request #19165 from makortel/phase2RemovePixelTracks

Remove pixel tracking from phase2

boudoul committed:
cleaning left over Phase2ITPixelClusterizer in python

jshlee committed:
davidlange6 suggestions

dimattia committed:
Fixing the statistics for MultiRunHarvesting

Sam-Harper committed:
changing to stream module to make thread safe

cmsbuild committed:
Merge pull request #19386 from franzoni/HPbyRun-DropBoxMetadata-from-CMSSW_9_2_3_patch1_reprise

DropBox Metadata: fix SiStripApvGain and add SiStripApvGainRcdAAG_multirun

cmsbuild committed:
Merge pull request #19252 from bpf6qc/METplusTrackTriggerOfflineDQM92X

Update to 2017 METplusTrack paths

cmsbuild committed:
Merge pull request #19392 from sarafiorendi/l2Tol1matching

Add parameter to disable L2 to L1 match

cmsbuild committed:
Merge pull request #19394 from Teemperor/FixTypoInOrPairSelector.h

Fix typo in OrPairSelector.h

cmsbuild committed:
Merge pull request #19397 from boudoul/cleaningphase2clustersIB

cleaning left over Phase2ITPixelClusterizer in python

cmsbuild committed:
Merge pull request #18560 from dildick/from-CMSSW_9_1_X_2017-04-28-1100-add-tmb-base-class

CSC Upgrade TMB base classes and improved LUT generator code

Commit:
Declare some of the arguments in function calls as const

Commit:
Try to conform with coding convention

nsmith- committed:
Add HLT_Physics trigger filter to L1T DQM

Replacing HLT_L1FatEvents since it is redundant to HLT_Physics

cmsbuild committed:
Merge pull request #19368 from ianna/geom-det-unit-cleanup

Remove Legacy Files

alesaggio committed:
Deprecation of T**DetId in DQM/SiStripCommissioningSources

rekovic committed:
Remove remaining L1TCalorimeter data files.

mmusich committed:
remove classes that need rethinking of access to TrackerTopology

cmsbuild committed:
Merge pull request #19376 from pieterdavid/removeSiStripSubDetId_RecoGeometryESProducts

Remove tracker subdetector DetId in some Reco/Geometry ES products

cmsbuild committed:
Merge pull request #19231 from davidlt/fix-calib-cond-operator

Fix || operator usage in conditional statements

Commit:
use empty()

Commit:
use empty()

Dr15Jones committed:
Allow specifying split level on a per branch basis

The new parameter overrideBranchesSplitLevel can be used to specify branch by branch what split level to use. This only works for data products made in this job, not products read from the input.

cmsbuild committed:
Merge pull request #19254 from dildick/from-CMSSW_9_2_X_2017-06-15-1100-me0-pad-digi-cluster

New feature in ME0 simulation: ME0PadDigiCluster

dimattia committed:
first bunch of comments implemented

cmsbuild committed:
Merge pull request #19410 from Sam-Harper/PMVarProdThreadSafeFix

making EgammaHLTPixelMatchVarProducer thread safe

cmsbuild committed:
Merge pull request #19408 from Dr15Jones/perBranchSettingOfSplitLevel

Allow specifying split level on a per branch basis

cmsbuild committed:
Merge pull request #19256 from bsunanda/Run2-hcx136

Run2-hx136 To enable access front layer position

davidlange6 committed:
quick noisy module

cerminar committed:
add a new alcareco to have the needed RAW from test enable in the alcaskimmign step

cerminar committed:
add the EcalTestPulsesRaw so that it is used but not persisted

cerminar committed:
fix the alcaskimming step for this scenario equipping it with the needed features for PCL

cerminar committed:
equip this sequence to use onky the data selected by the EcalTestPulsesRaw alcareco

cerminar committed:
add the DQMEventInfo to have basic information displayed in DQM GUI

cerminar committed:
adapt the relval 1010 to the new design for ecal pedestal PCL workflow

davidlange6 committed:
and for harvesting

cmsbuild committed:
Merge pull request #19194 from fioriNTU/TrackCutPhase1_92X

adding a simple track cut to all plugins using tracking

cmsbuild committed:
Merge pull request #19301 from threus/gtonlinev4_92x

update GT for online DQM to v4

cmsbuild committed:
Merge pull request #19257 from lantone/DisplacedSUSYTriggerOfflineDQM92X

Displaced SUSY Trigger Offline Monitoring for 2017

cmsbuild committed:
Merge pull request #19310 from cms-l1t-offline/PR92x-l1t-integration-v95.15

Pr92x L1T Calo (EG, MET, Tau)  2017 emulation of startup (up to l1t integration v95.15)

cmsbuild committed:
Merge pull request #19400 from dimattia/FixForMRH

Fixing the statistics for MultiRunHarvesting

dmitrijus commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

Commit:
Resolve Merge conflict

dmitrijus commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next master IBs after it passes the integration tests. This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @smuzaffar

davidlange6 committed:
replace invalid return condition by an assert

cmsbuild commented:
-1

Tested at: 7977fde8da7d197c260bb982ba64c5d2a87dbb41


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/5ad2ca330590b7d14a64a053f7f4350de7357460
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19391/20878/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19391/20878/git-merge-result

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19391/20878/summary.html

I found follow errors while testing this PR

**Failed tests:  UnitTests  RelVals**

* **Unit Tests**:

I found errors in the following unit tests: 
 
 

* **RelVals**:

The relvals timed out after 2 hours.



The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/5ad2ca330590b7d14a64a053f7f4350de7357460
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19391/20878/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19391/20878/git-merge-result

cmsbuild commented:
Comparison not run due to runTheMatrix errors (RelVals and Igprof tests were also skipped)

davidlange6 committed:
now with informative message

cmsbuild committed:
Merge pull request #19411 from davidlange6/nobtagvalid

quiet noisy module : hltBtagTriggerSelection

slava77devel committed:
make method buildSegments const and move all varying data members to an AlgoState percolated through all methods. This is a somewhat mindless method to make each call independent and resolve the problem with reproducibility running the algorithm in multithreaded mode or otherwise reordered events.

davidlange6 committed:
Revert "adding a simple track cut to all plugins using tracking"

cmsbuild committed:
Merge pull request #19420 from davidlange6/assertCond

Assert if formula is missing in PF calib

cmsbuild committed:
Merge pull request #19389 from z4027163/L1TDQMOffline_CMSSW_9_2_X

Remove stage2 modules from Legacy L1Trigger DQMOffline Sequence

cmsbuild committed:
Merge pull request #19385 from mmusich/92X_addEventInfoInAlCaPrompt

[92X] Adding the DQMEventInfo module in SiStripGains and SiPixelAli PCL sequences

cmsbuild committed:
Merge pull request #19311 from stahlleiton/DQMOnline_L1TMuon_QTsTune_v20170619

OnlineDQM : Tune L1T Muon Mismatch Ratio QTs

cmsbuild committed:
Merge pull request #19193 from mmusich/92X_SiStripPayloadsInpector

[92X] Add Payload Inspector classes for SiStripApvGains

cmsbuild committed:
Merge pull request #18950 from jalimena/NoBPTXTriggerOfflineDQM92X

add offline DQM for NoBPTX EXO triggers

cmsbuild committed:
Merge pull request #19422 from cms-sw/revert-19194-TrackCutPhase1_92X

Revert "adding a simple track cut to all plugins using tracking"

cmsbuild committed:
Merge pull request #19412 from cerminar/92X_EcalPedestalsConfigDP

[92x] EcalPedestal PCL workflow: fixes and improvements

cmsbuild committed:
Merge pull request #19197 from fabozzi/from-CMSSW_9_2_X_premix2017

adding premixing PU 2017 workflows

cmsbuild committed:
Merge pull request #19147 from bsunanda/Phase2-hgx84

Phase2-hgx84 Add functionality to HGCal geometry to access information

cmsbuild committed:
Merge pull request #18625 from jshlee/GEMSeeding-91x

seeding with ME0 Segments

cmsbuild committed:
Merge pull request #19421 from slava77/CMSSW_9_2_3_patch1/sign935/cscRUreproducible

make RU CSC segment algorithm reproducible by enforcing constness

cmsbuild committed:
Merge pull request #19405 from alesaggio/removeSiStripSubDetId_DQM_SiStripCommissioningSources

Deprecation of T**DetId in DQM/SiStripCommissioningSources

cmsbuild committed:
Merge pull request #19402 from nsmith-/hltPhysicsFatEvent

Add HLT_Physics trigger filter to L1T DQM

cmsbuild committed:
Merge pull request #19189 from jaylawhorn/RazorTriggerDQM_92X_PR

Razor trigger offline dqm in 92X

cmsbuild committed:
Merge pull request #18518 from cms-btv-pog/DQM_eta_phi_PR_from-CMSSW_9_1_X

B-tag DQM/Val.: Add efficiency plots vs. eta and phi for cut on discriminators

cmsbuild committed:
Merge pull request #19258 from bsunanda/Run2-sim09

Run2-sim09 Record particle flux in selected volumes

davidlange6 committed:
Revert "Update Phase 1 pixel gains in digitizer/clusterizer"

cmsbuild committed:
Merge pull request #19423 from cms-sw/revert-19181-92X_Update_Phase1_Pixel_Gains

Revert "Update Phase 1 pixel gains in digitizer/clusterizer"

cmsbuild committed:
Merge pull request #19216 from slehti/HLTTauValidationUpdate_CMSSW92X_June2017

Hlt tau validation update cmssw92 x june2017

Wilsker committed:
Fixed crash after rebase. Cleaned up comments.

Wilsker committed:
Fixed whitespace conflicts

cmsbuild commented:
Pull request #19391 was updated. @ghellwig, @lveldere, @ianna, @kpedro88, @fabozzi, @Martin-Grunewald, @rekovic, @vanbesien, @perrotta, @civanch, @monttj, @silviodonato, @cmsbuild, @fwyzard, @GurpreetSinghChahal, @davidlange6, @smuzaffar, @Dr15Jones, @kkousour, @mdhildreth, @cerminar, @slava77, @ggovi, @mulhearn, @ssekmen, @vazzolini, @kmaeshima, @arunhep, @dmitrijus, @franzoni can you please check and sign again.

perrotta commented:
-1
(it needs a rebase)
